File: src/question/day0706/ms_0706.cs
using System;

namespace algorithm_06
{
    class ms_0706
    {
        static void Main(string[] args)
        {
            int year = Convert.ToInt32(Console.ReadLine());

            if((year%4 == 0 && year%100 != 0) || year%400 == 0)
                Console.WriteLine("leap year");
            else 
                Console.WriteLine("common year");
        }
    }
}

File: src/question/month07/day0706/ms_0706.cpp
#include <iostream>

using namespace std;

int main() {
	int year;

	cin >> year;

	if (((year % 4 == 0) && (year % 100 != 0)) || (year % 400 == 0))
		cout << "leap year" << endl;
	else 
		cout << "common year" << endl;

	return 0;
}